pkg/yum/repository.go

```diff
@@ -158,7 +160,13 @@ func (r *Repository) Repomd() (*Repomd, int, error) {
 	if repomdURL, err = r.getRepomdURL(); err != nil {
 		return nil, 0, fmt.Errorf("Error parsing Repomd URL: %w", err)
 	}
-	if resp, err = r.settings.Client.Get(repomdURL); err != nil {
+
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, repomdURL, nil)
+	if err != nil {
+		return nil, 0, fmt.Errorf("error creating requesting: %w", err)
```

Comment:
nitpicky but there's a small typo in the error message here

Reply:
thanks!